code/data_structures/src/stack/stackUsingLinkedList/stackUsingLinkedList.cpp

```diff
@@ -0,0 +1,90 @@
+#include<iostream>
+#include<malloc.h>
+using namespace std;
+struct node
+{
+    int data;
+    struct node *next;
+};
+void Traverse_stack(struct node* top)
```

Comment:
fix names. Should all be lower camel case.

Reply:
okay..